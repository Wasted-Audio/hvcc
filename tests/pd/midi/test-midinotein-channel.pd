#N canvas 572 166 503 302 12;
#X obj 97 132 print;
#X obj 97 61 notein 2;
#X obj 97 96 pack f f;
#X connect 1 0 2 0;
#X connect 1 1 2 1;
#X connect 2 0 0 0;
