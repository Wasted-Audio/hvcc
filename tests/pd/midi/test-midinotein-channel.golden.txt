[@ 0.000] print: 48 64
[@ 0.000] print: 48 0
[@ 0.000] print: 60 64
[@ 0.000] print: 60 0
[@ 0.000] print: 64 64
[@ 0.000] print: 64 0
[@ 0.000] print: 60 64
[@ 0.000] print: 60 0
[@ 0.000] print: 65 64
[@ 0.000] print: 65 0
[@ 0.000] print: 60 64
[@ 0.000] print: 60 0
[@ 0.000] print: 64 64
[@ 0.000] print: 64 0
[@ 0.000] print: 60 64
[@ 0.000] print: 60 0
[@ 0.000] print: 62 64
[@ 0.000] print: 62 0
[@ 0.000] print: 59 64
[@ 0.000] print: 59 0
[@ 0.000] print: 60 64
[@ 0.000] print: 60 0
[@ 0.000] print: 57 64
[@ 0.000] print: 57 0
[@ 0.000] print: 53 64
[@ 0.000] print: 53 0
[@ 0.000] print: 55 64
[@ 0.000] print: 55 0
[@ 0.000] print: 48 64
[@ 0.000] print: 48 0